src/collection-facets/more-facets-content.ts

```diff
@@ -397,26 +397,21 @@ export class MoreFacetsContent extends LitElement {
   }
 
   private get getModalHeaderTemplate(): TemplateResult {
-    const title =
-      this.sortedBy === 'alpha' ? 'Sort by count' : 'Sort by alphabetically';
-
-    const image =
-      this.sortedBy === 'alpha'
-        ? 'https://archive.org/images/filter-alpha.png'
-        : 'https://archive.org/images/filter-count.png';
+    const title = this.sortedBy === 'alpha' ? 'Sort by count' : 'Sort by value';
 
     return html`<span class="sr-only">More facets for:</span>
       <span class="title">
         ${this.facetGroupTitle}
-        <input
-          class="sorting-icon"
-          type="image"
-          @click=${() => this.sortFacetAggregation()}
-          src="${image}"
-          title=${title}
-          alt="sort facets"
-        />
-      </span> `;
+        <a
```

Comment:
q: can we use button since it isn't really linking out to anywhere?

Reply:
Thanks for the suggestion! Agreed a button is more semantically accurate here -- updated.